### ISSUE TITLE
Feature Request/Enhancement:  Export progress  XX/XX or count down

### ISSUE THREAD

**Issue by dougK44883** [jessedp/tablo-tools-electron#172]
**Is your feature request related to a problem? Please describe.**
Previous versions would export sequentially.  When you have several screen fulls of shows, you could PageDown to find if/when it was done... or how much longer it had.  

Now, it seems exports are more randomly.  It's unclear when export is done (not true, cancel disappears) or how much longer it may have.  If you have several screens full, it's not readily clear.
 
**Describe the solution you'd like**
Ideally, I'd like to glance at the screen and have a clue how processing is progressing.

**Describe alternatives you've considered**
Possible suggestions:  7 of 25 complete  or number to process 18.  Obviously there's a big variable time for each export, so it's not an estimated time.  (although averaging times from different channels I suppose you could calculate time, but it's overkill)

**Additional context**
At times I let show accumulate, recording entire series.   Maybe more than one.  I've often export 70-100 shows at a time.  With the random order, it's not clear where the end is.

**Comment by jessedp** [comment]
I'll work on this.

I'm actually commenting to see if you'd noticed that they _"unlisted"_  my [post](https://community.tablotv.com/t/unofficial-tablo-api-docs-info/30751) on their forums linking to my [*Unofficial Tablo API..."](https://jessedp.github.io/tablo-api-docs/).

**Comment by dougK44883** [comment]
I did bookmark just in case so I could keep track.  I do know it can happen - I had one before - https://community.tablotv.com/t/3rd-party-apps-for-commerical-skip/22445  at one time they edited the playlist for com skip and I found it could be used to export... a commercial free recording.  They since changed their method.  

Recently some buffoon went off on some rant and was closed off in an hour or so.  When your's lasted a day I figured it was going to stay.  Really, many of the topics "in the early days" talk about all these things.  It's not hard to find.
